manager/integration/tests/test_recurring_job.py

```diff
@@ -476,13 +479,96 @@ def test_recurring_jobs_for_detached_volume():  # NOQA
     13. Verify that during the recurring backup, the volume's frontend is
         disabled, and pod cannot start.
     14. Wait for the recurring backup finishes.
-        Delete the recurring backup
-    15. In a 6-mintute retry loop, verify that the pod can eventually start.
-
-    16. Change the setting allow-recurring-job-while-volume-detached to false
-    17. cleanup
+        Delete the recurring backup.
+    15. In a 6-minutes retry loop, verify that the pod can eventually start.
+    16. Change the setting allow-recurring-job-while-volume-detached to false.
+    17. Cleanup.
     """
```

Comment:
Can we set Backupstore here
```
set_random_backupstore(client)
```
Without this, the test will fail when it is run separately since it cannot find a backup store.

Reply:
Addressed.